spec/models/amusement_park_spec.rb

```diff
@@ -4,4 +4,22 @@
   describe "relationships" do
     it { should have_many(:rides) }
   end
-end
\ No newline at end of file
+
```

Comment:
If you add an additional association to help with US 3, you could add here another association test:
`it { should have_many(:mechanics).through(:rides) }`